### ISSUE TITLE
Execution failed for task ':package_info_plus:compileDebugJavaWithJavac'. > error: invalid source release: 17

### ISSUE THREAD

**Issue by antoniofai** [FlutterFlow/flutterflow-issues#5111]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

i am try to deploy the app to the android simulator, the issue is appear.

### Expected Behavior

deploy to the android simulator

### Steps to Reproduce

You can access the project called ricefood in my account
Open the local run and deploy to the android simulator.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4khsrlx5NgoddG1s7pbcIxlj8vHFwhTLJMt9VxSUghfbXnEZoidffRT1VLYteRfH5beFWgilwd/sqKvYSeMME6JRGCft1MzL5XQQrKJHK7Woy2PM6/OGh6I8FTIUyD352JgyNdIrFacUIh6kyuefCrG3qCf9qOYwx5e6fDbOY=

### Visual documentation

Launching lib\main.dart on sdk gphone x86 in debug mode...
Upgrading build.gradle
Running Gradle task 'assembleDebug'...                          

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':package_info_plus:compileDebugJavaWithJavac'.
> error: invalid source release: 17

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 43s
Running Gradle task 'assembleDebug'...                             44.2s
Error: Gradle task assembleDebug failed with exit code 1

### Environment

```markdown
- FlutterFlow version: 5.0.13
- Platform: windows, windows flutterflow app
- Browser name and version: using the flutterflow window app
- Operating system and version affected: Windows 10
```

**Comment by Alezanello** [comment]
Hello!

It looks like your local machine is running Java 17, but some packages, including this one, require Java 18 to compile. Please update your Java version to 18 to resolve this issue.

**Comment by antoniofai** [comment]
can i ask the local machine is mean the computer or the android simulator ? Thanks for your reply,

**Comment by Alezanello** [comment]
The java version of your computer must be Java 18 to run FF projects!

**Comment by antoniofai** [comment]
Ok, let me try first, and come back later, thx🫡

**Comment by antoniofai** [comment]
I already have the java version 20 development kit, but the error still happen.

**Comment by Alezanello** [comment]
That's correct because you need Java 18 like mentioned before for the package_info_plus

**Comment by greghawk** [comment]
Yeah well flutterflow local for android forces gradle-7.5 which doesn't work with 18?

**Comment by Alezanello** [comment]
Could you please run the `flutter doctor -v` command in your terminal and share the results with us? This will help us better understand your setup and assist you more effectively.

**Comment by Alezanello** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Thank you!
